README.md

```diff
@@ -2,10 +2,10 @@
 
 [baustelle.live](https://baustelle.live) is a hobby project by [Felix De Montis](https://felix.dm) / @dervondenbergen.
 
-One of the construction sites of a new U2 subway station is since 2020 up until 2028 in the street crossing Lindengasse/Kirchengasse in Vienna's 7th district.
+One of the construction sites of a new U2 subway station is up since 2020 until 2028 in the street crossing Lindengasse/Kirchengasse in Vienna’s 7<sup>th</sup> district.
```

Comment:
This `up` was used here in conjunction with the `until`, like _bis zum (jahr)_ instead of only _bis_.

Reply:
In this case, I would write “One of the construction sites of a new U2 subway station is situated in the street crossing Lindengasse/Kirchengasse in Vienna’s 7<sup>th</sup> district since 2020 up until 2028.”

Otherwise it looks a bit odd when there is just the verb “is”.